packages/components/src/mobile/bottom-sheet/index.native.js

```diff
@@ -262,10 +261,7 @@ class BottomSheet extends Component {
 			//	On vertical mode `maxHeight` has to be set on 50% of width
 		} else {
 			this.setState( {
-				maxHeight: Math.min(
```

Comment:
We probably only need to check the min if `isFullScreen` is false. 
Other wise we need to figure out what the fullscreen height should be